### PR TITLE
feat: capture customer photo during tx

### PR DESCRIPTION
**Pull request by josepfo**

### CODE REVIEW

**Comment by josepfo** [comment]
[Automatically capture customer photo upon acceptance of Terms & Conditions](https://trello.com/c/pjmajldc/890-automatically-capture-customer-photo-upon-acceptance-of-terms-conditions)

**Review by chaotixkilla — APPROVED** [review]

File: lib/brain.js [hunk 1]
```diff
@@ -2132,11 +2128,13 @@ Brain.prototype.mustAcceptTerms = function mustAcceptTerms () {
...
-  // and redirect user to start screen
-  this.tx = Tx.update(this.tx, { termsAccepted: true })
-  this.startScreen()
+  if (this.scanner.isOpened()) this.scanner.cancel()
+  setTimeout(() => {
+    // mark terms as accepted
+    // and redirect user to start screen
+    this.tx = Tx.update(this.tx, { termsAccepted: true })
+    this.startScreen()
+  }, 100)
```
Comment (chaotixkilla) [inline lib/brain.js:2137]: A bit confused on why this timeout is needed, could you explain?

**Review by chaotixkilla — APPROVED** [review]

File: lib/brain.js (hunk 1 above)
Comment (RafaelTaranto) [reply lib/brain.js:2148]: can you add a comment explaining the timeout?

File: lib/brain.js [hunk 2]
```diff
@@ -2553,31 +2568,34 @@ Brain.prototype._handleScan = function _handleScan (address) {
...
-      if (it.blacklisted) {
-        return setTimeout(() => {
-          this._timedState('suspiciousAddress')
-        }, remaining)
-      }
-
-      setTimeout(() => {
-        return this._firstBill()
-      }, remaining)
+      this.updateTxCustomerPhoto(it)
```
Comment (RafaelTaranto) [inline lib/brain.js]: Can't we trigger this without expecting the response?

**Review by RafaelTaranto — CHANGES_REQUESTED** [review]

**Review by RafaelTaranto — APPROVED** [review]

**Pull request merged**